packages/app/src/contacts/presenters/contact.presenter.test.ts

```diff
@@ -14,6 +14,7 @@ const pureContact: PureContact = {
   blocked: false,
   favourite: true,
   numbers: ["500400300"],
+  numbersIDs: ["1"],
```

Comment:
Please keep name conventions with the current project or rename it for all 😉 
```suggestion
  numberIds: ["1"],
```

The current state of the project
```
id -> // bad ID
ids -> // bad IDs
numberIds  // bad -> numbersIDs / numberIDs
numberId  // bad -> numberID

```

Reply:
packages/app/src/device/types/mudita-os/serialport-request.type.ts

here are interfaces PureMessage, PureThread and PureContact - we probably are keeping this convention 'ID' because of Pure, we can change it, sure, but in my opinion it would add confusion to the code like:
- we receive data as 'ID', then we rename it to 'Id', next we use it

and i agree that name 'numbersIDs' might not be the best :D